File: section-03-basic-maths/01-count-all-digits-of-a-number/count-all-digits-of-a-number-opt.cs
using System;

public class Solution {
    /* Function to count the 
    number of odd digits in N */
    public int countDigit(int n) {
        // Edge case
        if (n == 0) return 1;
        
        int count = (int)Math.Floor(Math.Log10(n) + 1);
        return count;
    }
}

public class Program {
    public static void Main(string[] args) {
        // Input number
        int n = 6678;

        // Creating an instance of Solution class
        Solution sol = new Solution();

        // Function call to get count of digits in n
        int ans = sol.countDigit(n);
        Console.WriteLine("The count of digits in the given number is: " + ans);
    }
}

File: section-03-basic-maths/01-count-all-digits-of-a-number/count-all-digits-of-a-number-opt.py
import math

class Solution:
    # Function to count the
    # number of odd digits in N
    def countDigit(self, n):
        # Edge case
        if n == 0:
            return 1
            
        count = int(math.log10(n) + 1)
        return count

# Input number
n = 6678

# Creating an instance of Solution class
sol = Solution()

# Function call to get count of digits in n
ans = sol.countDigit(n)
print("The count of digits in the given number is:", ans)